package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/ionide/ionide-vscode-helpers#readme",
   "scripts": {
+    "gen:vscode": "ts2fable node_modules/@types/vscode/index.d.ts src/Fable.Import.VSCode.fs"
```

Comment:
maybe we could add something like `"gen:vscode-proposed": "vscode-dts dev && ts2fable (whatever-is-the-name-of-the-file-i-dont-remember.d.ts) src/Fable.Import.VSCode.Proposed.fs`

Reply:
certainly! I'm currently doing a WIP conversion of ionide to use these new generated helpers to make sure things don't puke too badly, so once I've done that to validate this core idea I'd be glad to add more

Reply:
sidenote: Thenables are a pain in the ass to work with in Fable.  I need a way to lift them to the Promise<'t> shape.